peytonjo/Dad-Jokes
# Fix/clean up

### What does this PR do?
- Refactors files 
- Adds scss fixes
- Fixes indents

### Where should the reviewer start?
- line 1 of all files 
- all files have been changed

### How should this be manually tested?
- run npm test 
- npm start 
- this way you can see all tests are passing and see the application itself 

### Any background context you want to provide?
- N/A
